Setup api (#206)
GRAAL-Research/deepparse: README.md

docker compose up app
```

> #### Sentry:
> Also, you can monitor your application usage with [Sentry](https://sentry.io) by setting the environment variable `SENTRY_DSN` to your Sentry's project
> DSN. There is an example of the .env file in the root of the project named `.env_example`. you can just copy it using the following command: 
> 
> ```shell
> cp .env_example .env
> ```
> 
> If you do not have a Sentry account, you can create one [here](https://sentry.io/signup/).
> 
> If you do not want to use Sentry, you can just remove the `SENTRY_DSN` environment variable from the `.env` file or set it to an empty string, also, not creating
> the `.env` file will also work. The application will run without any problem if Sentry is not set.

Reviewer: ```suggestion
#### Sentry:
Also, you can monitor your application usage with [Sentry](https://sentry.io) by setting the environment variable `SENTRY_DSN` to your Sentry's project
DSN. There is an example of the .env file in the project's root named `.env_example`. You can copy it using the following command: 

```shell
cp .env_example .env
```

If you do not have a Sentry account, you can create one [here](https://sentry.io/signup/).

If you do not want to use Sentry, you can remove the `SENTRY_DSN` environment variable from the `.env` file or set it to an empty string; also, not creating
the `.env` file will work. The application will run without any problem if Sentry is not set.
```
